src/app/components/chat/chat.component.html

```diff
@@ -1 +1,17 @@
-<p>chat works!</p>
+<div class="chat-window">
+  @if (messages$ | async; as messages) {
+  <div id="messages">
+    @for (message of messages; track $index) { @if (message.userId) {
+    <div>{{ message.userId }} : {{ message.messageText }}</div>
+    }@else {
+    <div>{{ message.messageText }}</div>
+    } }
+  </div>
+  }
+
+  <div id="users"></div>
+</div>
+<form class="chat-box" id="form" (ngSubmit)="sendMessage()">
+  <input id="input" name="text" placeholder="Message" />
+  <button type="submit">Submit</button>
```

Comment:
Just a suggestion, perhaps the button could display "Send" instead of "Submit"?

Reply:
Or even just the send message icon
e.g. https://react-icons.github.io/react-icons/search/#q=send

Reply:
Okay cool, thanks i changed it in the UI branch